c++/include/orc/Writer.hh

```diff
@@ -277,6 +277,17 @@ namespace orc {
      * @return if not set, return default value which is 1 MB.
      */
     uint64_t getOutputBufferCapacity() const;
+
+    /**
+     * Set the initial block size of input buffer in the class CompressionStream.
```

Comment:
Could we add more comment to let readers know what's the difference with `setOutputBufferCapacity` above? Perhaps we also need to modify the comment in setOutputBufferCapacity. Users rely on these public methods to tune their use cases.

Reply:
Thanks for the reminder, more comments have been added here.